guacamole/src/main/webapp/app/rest/types/Connection.js

```diff
@@ -104,6 +104,15 @@ angular.module('rest').factory('Connection', [function defineConnection() {
          */
         this.sharingProfiles = template.sharingProfiles;
 
+        /**
+         * The time that this connection was last used, in seconds since
+         * 1970-01-01 00:00:00 UTC. If this information is unknown or
+         * unavailable, this will be null.
+         *
+         * @type Number
```

Comment:
Based on the API code above on the servlet side, it looks like this parameter is delivered by Java as a Date/Time, but is a number of seconds (epoch) here.  Does the API deliver it as a number of seconds, or is this translation done automatically?

Reply:
@mike-jumper Any insight on this one?

Reply:
Whoops - didn't see this comment. Yes, the Java `Date` is automatically converted to a timestamp in seconds by Jersey+Jackson when the object is serialized to JSON.

Reply:
Actually, I may be wrong here ... all the documentation I'm able to find thus far indicates that Jackson serializes to milliseconds, not seconds. @jmuehlner - do you recall offhand?

Reply:
Okay...I'll hold off on the merge until we nail that down.

Reply:
It looks like milliseconds, yeah. The documentation indicates so, as does inspection of existing API responses.